pyaedt/modules/SetupTemplates.py

```diff
@@ -1329,29 +1330,50 @@ class SweepHFSS(object):
 
     Parameters
     ----------
-    oanalysis :
-
+    app : :class 'pyaedt.modules.SolveSetup.Setup'
+        The setup used for the analysis.
```

Comment:
```suggestion
        Setup used for the analysis.
```

Reply:
Way back, Alex and I decided we'd not use articles so that every description didn't start with "The," "A," or "An".

Reply:
Thanks @PipKat . I will keep that in mind.